### DISCUSSION TITLE
Configuration Option lock / unlock debug

### DISCUSSION THREAD

**Discussion by Corran1327** [Courseplay/courseplay#7597] [Q&A] [answered]
So to start my understanding is that the ModHub version and the GitHub Version are different and for a relatively good reason

Github is the Only version that has the Debug modes enabled.

I think it would be great if this functionality was Locked in the Modhub version but would be able to be Unlocked in the advanced config panel.

Thanks for everything you guys do

**Comment by Tensuko** [comment, marked as answer]
No reason to do so.
GitHub Version is the one that needs debugging.
Any 'Error' from ModHub will be ignored anyway, becouse it is way to old.
